src/registrar/templates/domain_your_contact_information.html

```diff
@@ -7,7 +7,8 @@
 
   <h1>Domain contact information</h1>
 
-  <p>If you’d like us to use a different name, email, or phone number you can make those changes below. Changing your contact information here won’t affect your Login.gov account information.</p>
+<p>If you’d like us to use a different name, email, or phone number you can make those changes below. <strong>Please note that updating your contact information here will update the contact information for all domains in your account.</strong> However, it won’t affect your Login.gov account information.
```

Comment:
Non-blocking comment: there's a part of me that dislikes seeing "please" in text on websites. Especially when coming from a government, something about it feels patronizing. Consider dropping "Please note that", but I defer to you.

Reply:
^^ I agree with this, I think in this context not having that would be best

Reply:
Great suggestion. @Katherine-Osos has updated the content guide to note this for future reference.